libs/cimit-service/src/main/java/uk/gov/di/ipv/core/library/service/CiMitService.java

```diff
@@ -72,72 +104,137 @@ public CiMitService(
         this.lambdaClient = lambdaClient;
         this.configService = configService;
         this.verifiableCredentialValidator = verifiableCredentialValidator;
+        this.httpClient = HttpClient.newHttpClient();
+    }
+
+    @ExcludeFromGeneratedCoverageReport
+    public CiMitService(
+            LambdaClient lambdaClient,
+            ConfigService configService,
+            VerifiableCredentialValidator verifiableCredentialValidator,
+            HttpClient httpClient) {
+        this.lambdaClient = lambdaClient;
+        this.configService = configService;
+        this.verifiableCredentialValidator = verifiableCredentialValidator;
+        this.httpClient = httpClient;
     }
 
+    @Tracing
     public void submitVC(VerifiableCredential vc, String govukSigninJourneyId, String ipAddress)
             throws CiPutException {
 
-        String payload;
+        LOGGER.info(LogHelper.buildLogMessage("Sending VC to CIMIT."));
         try {
-            payload =
-                    OBJECT_MAPPER.writeValueAsString(
-                            new PutCiRequest(govukSigninJourneyId, ipAddress, vc.getVcString()));
-
+            if (configService.enabled(CIMIT_API_GATEWAY_ENABLED)) {
+                var payload =
+                        OBJECT_MAPPER.writeValueAsString(
+                                new PostCiPrivateApiRequest(vc.getVcString()));
+
+                var uri = getUriBuilderWithBaseApiUrl(POST_CI_ENDPOINT).build();
+
+                var response = sendPostHttpRequest(uri, govukSigninJourneyId, ipAddress, payload);
+                var parsedResponse =
+                        OBJECT_MAPPER.readValue(response.body(), PrivateApiResponse.class);
+
+                if (FAILED_RESPONSE.equals(parsedResponse.result())) {
+                    logApiRequestError(parsedResponse);
+                    throw new CiPutException(FAILED_API_REQUEST);
+                }
```

Comment:
Could parsing the response and detecting failures live inside `sendPostHttpRequest` (perhaps renamed to `sendCimitApiRequest`)? Looks the same for each method